modules/ROOT/pages/data-modeling.adoc

```diff
@@ -62,11 +58,17 @@ This rule applies to writes performed through:
 When you might encounter the error::
 You may encounter the error in the following deployment situations:
 
-* In Mobile-to-Web Data Sync with Field-level Encryption enabled, because the rule conflicts with the default {xref-sdk-java-encryption}
 * In Mobile-to-Web Data Sync with {xref-sdk-nodejs-pg-start} and http://ottomanjs.com/[Ottoman.js] (the Node.js ODM for Couchbase), where the rule conflicts with the `+_type+` property that is automatically added by _Ottoman.js_.
 +
 A suggested workaround in this scenario is to fork the _Ottoman.js_ library, perform a search-replace for the `+_type+` property and replace it without a leading underscore.
 
+[NOTE] 
+====
+For versions 2.x of Sync Gateway, you can encounter the following error:
+
+* In Mobile-to-Web Data Sync with Field-level Encryption enabled, because the rule conflicts with the default {xref-sdk-java-encryption}
```

Comment:
```suggestion
* In Mobile-to-Web Data Sync with Field-level Encryption enabled, because the rule conflicts with the default {xref-sdk-java-encryption}.
```

Reply:
Done